### DISCUSSION TITLE
Flutter framework

### DISCUSSION THREAD

**Discussion by Pl4tt** [community/community#9290] [Copilot Conversations]
# Flutter framework
when i write the following:
// an input page with the properties:
// grade(double), subjectId(int), weight(double)
// at the bottom there should be a submit button

i just get:
child: Center(
    child: Text("Create Grade"),
),

Do i need to write something different or is the copilot just not able to work with flutter?

**Comment by jaiswal4sudeep** [comment]
Copilot absolutely works with the flutter. You just need to write down the sufficient keywords for performing the task 👍
